SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Storage/StorageAdapter.cs

```diff
@@ -13,7 +13,21 @@ internal class StorageAdapter : IStorageAdapter
 
         public StorageAdapter(string databasePath)
         {
-            _dbConnection = new SQLiteAsyncConnection(databasePath);
+            // In SQLite-net 1.5.231 constructor parameters was changed.
+            // Using reflection to accept newer library version.
+            _dbConnection = (SQLiteAsyncConnection)typeof(SQLiteAsyncConnection)
+                .GetConstructor(new[] { typeof(string), typeof(bool) })
+                ?.Invoke(new object[] { databasePath, true });
```

Comment:
I don't see boolean parameter in https://github.com/oysteinkrog/SQLite.Net-PCL/blob/master/src/SQLite.Net.Async/SQLiteAsyncConnection.cs.

Previous version of the code has single parameter, in your patch you either pass 2 or 3 parameters (so more than 1 even in old version), I am confused about that, can you comment more about what code is for what version and what are the parameters for?

Reply:
You linked some fork where last update was in 2015.
The second parameter had default value. See https://github.com/praeclarum/sqlite-net/commit/c360ae602dd3a36d8ffdb60e3101c872a5de872a#diff-2388537c7df63c2e1c2c63b815f301b4L57